Add row enter/leave animation support via ReactCSSTransitionGroup

saary opened the pull request:
1. Added dependency on [rc-css-transition-group](https://github.com/react-component/css-transition-group) for proper React animation support.
2. Updated the docs accordingly.

saary committed:
support ReactCSSTransitionGroup for row enter animations. Update docs accordingly

saary committed:
support ReactCSSTransitionGroup for row enter animations. Update docs accordingly

kevinold commented:
@saary Please add tests before this can be officially approved and merged.  Thanks.

rashthedude commented:
What happened to this merge, any updates?

Commit:
Subgrid Icons are not Displayed if Metadata

Fix for the following issue:
The component or characters that are added into the first column of a subgrid row
(parentRowCollapsedComponent and parentRowExpandedComponent, default or custom) are
not displayed when a metadata object (from columnMetadata) exists for this first column.

saary committed:
Merge with Griddle main master

saary committed:
Use Object.create instead of extend to enjoy diff capabilities

jsdmc committed:
changed keys for rows to fix https://github.com/facebook/react/issues/2410

jsdmc committed:
no Data section moved inside table

jsdmc committed:
fixed error when rendering subgrids (due to unique keys)

joellanciaux committed:
Now hiding paging content if there's no data.

Commit:
add docs for custom filter function and component

waffle-iron committed:
add waffle.io badge

saary committed:
Add support for forceUpdate of columns

saary committed:
Add support to custome shouldUpdate metadata functions

saary committed:
Add modules to repo

html-monster commented:
https://github.com/html-monster/ReactUpdateAnimation
https://html-monster.github.io/ – Demo
I’ve modified an extension of ReactCSSTransitionGroup component. It expands component capabilities, now the class will be added not only while the appearance or removal of an element, but also when an item is updated
